go-gorilla/src/internal/handlers/api_handlers.go

```diff
@@ -0,0 +1,274 @@
+package handlers
+
+import (
+	"crypto/md5"
+	"encoding/json"
+	"fmt"
+	"go-gorilla/src/internal/auth"
+	"go-gorilla/src/internal/config"
+	"go-gorilla/src/internal/db"
+	"go-gorilla/src/internal/models"
+
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+func API_Follow(w http.ResponseWriter, r *http.Request) {
+	//Log
+	fmt.Println("Follow handler invoked")
```

Comment:
remove all print stuff that is not attached to errors. I think this can also be added in the wiki: requirements

Reply:
done